RecipeApi/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -20,6 +20,10 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
             {
                 await next.Invoke(context);
             }
+            catch(ForbidException forbidException){
+                context.Response.StatusCode = 403;
```

Comment:
Don't need to do that when using attributes.
Try to login as non admin and call admin method. The exception will be thrown by default.

Reply:
I add is there to catch my forbidException from every place where i throw that, when I start use standard exceptions it will be unnecessary.